behaviors/wysiwyg.js

```diff
@@ -270,6 +333,47 @@ function addComponent(el, text) {
 }
 
 /**
+ * add MULTIPLE components after the current component
+ * @param {Element} el
+ * @param {array} components (array of text models)
+ * @returns {Promise}
+ */
+function addComponents(el, components) {
+  var current = getCurrent(el),
+    parent = getParent(current.component);
+
+  // create the new components
+  return Promise.all(_.map(components, function (component) {
+    var throwawayDiv = document.createElement('div'),
+      fragment = model.toElement(component);
+
+    throwawayDiv.appendChild(fragment);
+
+    return edit.createComponent(current.name, {
+      text: throwawayDiv.innerHTML
```

Comment:
> If the element has no parent node, that is if it is the root node of the document, setting its outerHTML property will throw a DOMException with the error code NO_MODIFICATION_ALLOWED_ERR.

Yeah, it's not great, but apparently the temp div is the current best practice :disappointed: